lib/modules/storage/index.js

```diff
@@ -156,24 +156,22 @@ class Storage {
 
   checkStorageService(platform, url, callback) {
     const self = this;
-    const errorObj = {message: __('Cannot upload: {{platform}} node is not running on {{url}}.', {platform: platform, url: url})};
 
     // start the upload storage node
     self._checkStorageEndpoint(platform, function (err) {
       if (!err) {
-        return callback();
+        return callback(null);
```

Comment:
Why specify `null`? Empty should work.

Reply:
Fair point, most likely a hangover from when I was trying different async functions and passing values through the callbacks.